e2e/commands_test.go

```diff
@@ -19,6 +19,16 @@ import (
 	"gotest.tools/icmd"
 )
 
+func TestExitErrorCode(t *testing.T) {
+	cmd, cleanup := dockerCli.createTestCmd()
+	defer cleanup()
+
+	cmd.Command = dockerCli.Command("app", "unknown_command")
+	icmd.RunCmd(cmd).Assert(t, icmd.Expected{
+		ExitCode: 1,
+	})
```

Comment:
Should we also expect the error message?

Reply:
yes, done 😉